go/fn/examples/example_filter_GVK_test.go

```diff
@@ -22,7 +22,7 @@ func updateReplicas(rl *fn.ResourceList) error {
 	var replicas int
 	rl.FunctionConfig.GetOrDie(&replicas, "replicas")
 	for i, obj := range rl.Items {
-		if obj.APIVersion() == "apps/v1" && obj.Kind() == "Deployment" {
+		if obj.GetAPIVersion() == "apps/v1" && obj.GetKind() == "Deployment" {
```

Comment:
Is there specific reason for re-introducing `GetX()` ? the earlier versions looked good.

Reply:
yes, we want the KubeObject be used as unstructured.Unstrucutred to reduce the k8s users learning curve.  unstructured.Unstrucutred has functions GetKind, setKind, GetNamespace, SetNamespace
https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured